src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -1682,10 +1700,12 @@ static void lb_on_response_received_locked(grpc_exec_ctx *exec_ctx, void *arg,
             rr_handover_locked(exec_ctx, glb_policy);
           }
         } else {
-          if (GRPC_TRACER_ON(grpc_lb_glb_trace)) {
-            gpr_log(GPR_INFO,
-                    "Received empty server list. Picks will stay pending until "
-                    "a response with > 0 servers is received");
+          if (glb_policy->serverlist == NULL) {
```

Comment:
I think we should log a message when we get an empty serverlist, even if we have a previous serverlist.  It's just that the message should be different in each case.  If we do not have a previous serverlist, the existing message is appropriate.  If we do have a previous serverlist, then the message should say that we're continuing to use the previous serverlist.

Reply:
Accepted.